nomos-services/network/src/backends/libp2p.rs

```diff
@@ -112,14 +115,20 @@ impl NetworkBackend for Libp2p {
                                 log_error!(swarm.connect(peer_id, peer_addr));
                             }
                             Command::Broadcast { topic, message } => {
-                                match swarm.broadcast(&topic, message) {
+                                match swarm.broadcast(&topic, message.clone()) {
```

Comment:
it might be worth to change the network subscribtion system to use `Bytes` in case there are many subscribers but I don't think it makes any difference in this case because libp2p converts it into a vec anyway (and thus clone it)